build.xml

```diff
@@ -100,8 +94,13 @@
 		  	<replacefilter token="@CLOUD_SECRET@" property="user_input.CLOUD_SECRET"/>
 			<replacefilter token="@CLOUD_SHARED_BASE_PATH@" property="user_input.CLOUD_SHARED_BASE_PATH"/>
 			<replacefilter token="@LOG_DIR@" property="user_input.LOG_DIR"/>
+			<replacefilter token="@PLAY_SERVER_BASE_DIR@" property="user_input.PLAY_SERVER_BASE_DIR"/>
```

Comment:
we don't need PLAY_SERVER_BASE_DIR in user_input.ini, please change this line to:<replacefilter token="@PLAY_SERVER_BASE_DIR@" value="${basedir}"/>